crates/host/src/wasmbus/mod.rs

```diff
@@ -1443,7 +1445,14 @@ impl ActorInstance {
                 let target = invocation.target.clone();
                 let operation = invocation.operation.clone();
 
+                let start_at = Instant::now();
                 let res = self.handle_call(invocation).await;
+                let elapsed = u64::try_from(start_at.elapsed().as_nanos()).unwrap_or_default();
+
+                self.metrics
+                    .wasmcloud_host_handle_rpc_message_duration_ns
+                    .record(elapsed, &[KeyValue::new("operation", operation.clone())]);
```

Comment:
Can we also tag with the actor ID? I imagine being able to see duration by actor is more important than an aggregate of all workloads on the host

Reply:
We chatted offline, instead of logging the `actor_id`, we decided to log the `actor.ref` instead since the (at least, current) actor id is going away in 1.0.

We can revisit to add more things here before 1.0 ships if it makes sense, but for now we at least have a way to identify things by actor.ref for now.